modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -626,10 +781,7 @@ static void setROIBlob(InferenceEngine::InferRequest& req,
                        const IECallContext&           ctx) {
     if (ctx.uu.params.device_id.find("GPU") != std::string::npos &&
         ctx.uu.rctx) {
-        GAPI_LOG_WARNING(nullptr, "ROI blob creation for device_id: " <<
-                         ctx.uu.params.device_id << ", layer: " << layer_name <<
-                         "is not supported yet");
-        GAPI_Assert(false && "Unsupported ROI blob creation for GPU remote context");
+        setBlob(req, layer_name, blob, ctx);
```

Comment:
Why we ignore `roi` here, it will cause accuracy problems that will be difficult to debug